about.html

```diff
@@ -40,7 +40,7 @@
       <div class="nav-bar">
         <a href="#" aria-label="bars" class="menu" onclick="menu();"><i class="fa fa-bars"></i></a>
         <ul class="nav-bar-list">
-          <li><a href="home.html" class="nav-bar-item" onclick="closeMenu();">Home Page</a></li>
+          <li><a href="index.html" class="nav-bar-item" onclick="closeMenu();">Home Page</a></li>
```

Comment:
- [ ] Please link the about.html file with the index.html file properly, I am getting this error anytime I need to go back to the main page or access other pages, fix this before you submit.
- [ ] It seems like you are calling index.html file in your code but I see home.html in the browser.
![index](https://user-images.githubusercontent.com/55635977/124870310-c6fc3900-dfc2-11eb-8f2a-f9319e9057d1.png)